<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-98AE8SAQ" ignored="0" name="Retry Test" tests="5" failures="2" timestamp="2019-10-13T12:50:05 IST" time="0.061" errors="0">
  <testcase name="Test1" time="0.014" classname="retryLogic.myTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="retryLogic.myTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="retryLogic.myTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="retryLogic.myTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryLogic.myTests.Test1(myTests.java:12)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.001" classname="retryLogic.myTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryLogic.myTests.Test2(myTests.java:18)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Retry Test -->
